src/main/scala/org/opennetworkinsight/FlowPreLDA.scala

```diff
@@ -171,8 +173,22 @@ object FlowPreLDA {
           }
 
           logger.info("Trying to read file:  " + config.inputPath)
-          val rawdata: RDD[String] = sc.textFile(config.inputPath)
-          val datanoheader: RDD[String] = FlowWordCreation.removeHeader(rawdata)
+          val rawdata: RDD[String] = {
```

Comment:
okay
be sure to create a jira for it or add this as a subtask to the dataframe change